bean-validation/3.0/_index.md

```diff
@@ -1,25 +1,23 @@
 ---
-title: "Jakarta Bean Validation 3.0 (Milestone)"
-date: 2020-06-09
+title: "Jakarta Bean Validation 3.0"
+date: 2020-07-07
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Bean Validation defines a metadata model and API for JavaBean and method validation.
 
 * [Jakarta Bean Validation 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.bean-validation/releases/3.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Bean Validation 3.0 Specification Document](./bean-validation-specification-3.0.0-M1.pdf) (PDF)
-* [Jakarta Bean Validation 3.0 Specification Document](./bean-validation-specification-3.0.0-M1.html) (HTML)
+* [Jakarta Bean Validation 3.0 Specification Document](bean-validation-specification-3.0.pdf) (PDF)
+* [Jakarta Bean Validation 3.0 Specification Document](bean-validation-specification-3.0.html) (HTML)
 * [Jakarta Bean Validation 3.0 Javadoc](./apidocs)
-* [Jakarta Bean Validation 3.0 TCK](http://download.eclipse.org/ee4j/bean-validation/3.0/beanvalidation-tck-dist-3.0.0-M2.zip)([sig](),[sha](),[pub]())
-  * TCK Summary Results [Hibernate Validator 7.0.0.Alpha3](./TCK-summary.html)
+* [Jakarta Bean Validation 3.0 TCK](http://download.eclipse.org/ee4j/bean-validation/3.0/beanvalidation-tck-dist-3.0.0.zip)([sig](),[sha](bb0c85c6ab78786b023931cc407e00bbcd96fc26c0d8b56a00c28bc1665c7f01),[pub]())
```

Comment:
Just like you did for the DI PR, the TCK dl, sig, and pub items need to be filled in with the "final" values.